src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -1282,12 +1313,26 @@ private Task<INetworkInterface> AssociateNicWithNetworkSecurityGroupAsync(INetwo
                 () => networkInterface.Update().WithExistingNetworkSecurityGroup(networkSecurityGroup).ApplyAsync()
             );
 
-        private Task CreateMySqlDatabaseUser(ConnectionInfo sshConnectionInfo, Server mySqlServer)
+        private Task CreatePostgreSqlDatabaseUser(ConnectionInfo sshConnectionInfo)
             => Execute(
-                $"Creating MySQL database user...",
-                () => ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"mysql -u {configuration.MySqlAdministratorLogin} -p'{configuration.MySqlAdministratorPassword}' -h {mySqlServer.FullyQualifiedDomainName} -P 3306 < {CromwellAzureRootDir}/mysql-init/init-user.sql")
+                $"Creating PostgreSQL database user...",
+                () => ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"psql postgresql://{configuration.PostgreSqlAdministratorLogin}:{configuration.PostgreSqlAdministratorPassword}@{configuration.PostgreSqlServerName}.postgres.database.azure.com/{configuration.PostgreSqlDatabaseName} < {CromwellAzureRootDir}/mysql-init/init-user-postgre.sql")
             );
 
+        private Task<IPrivateDnsZone> CreatePrivateDnsZoneAsync(INetwork virtualNetwork)
+            => Execute(
+                "Creating private DNS Zone for PostgreSQL Server",
+                async () =>
+                {
+                    // Note: upon refactoring out of fluent see commit cbffa28 in #392. 
+                    var postgreSqlDnsZone = await azureSubscriptionClient.PrivateDnsZones
+                        .Define($"{configuration.PostgreSqlServerName}.private.postgres.database.azure.com")
+                        .WithExistingResourceGroup(configuration.ResourceGroupName)
+                        .DefineVirtualNetworkLink($"{virtualNetwork.Name}-link").WithReferencedVirtualNetworkId(virtualNetwork.Id).DisableAutoRegistration().Attach()
```

Comment:
Normally with the fluent style, each of the method calls is on its own line... I understand these are specific to the Virtual Network link, but I think typically these would still be each on its own line..  could be wrong, but thought I'd mention...

.WithReferencedVirtualNetworkId(virtualNetwork.Id).DisableAutoRegistration().Attach()

Reply:
agreed, put on separate lines.